### ISSUE TITLE
Argentina - October touristic public holiday missing

### ISSUE THREAD

**Issue by martingra** [nager/Nager.Date#578]
### Affected country

Argentina

### Affected public holiday

October touristic public holiday

### Source of the information

https://www.argentina.gob.ar/interior/feriados-nacionales-2024

### More information

There are 3 public holidays that are included into the calendar by law. This is one of those 3 (11 October)

**Comment by tinohager** [comment]
According to the legal text, up to 3 additional public holidays can be defined each year. However, I do not read from the text that these are fixed holidays. This means that they have to be added every year. Do I understand this correctly?

https://www.boletinoficial.gob.ar/detalleAviso/primera/301434/20231229

> WHEREAS, Law No. 27,399 empowers the NATIONAL EXECUTIVE BRANCH to set up to THREE (3) holidays or non-working days per year to promote tourism, which shall coincide with Mondays or Fridays.
--
Que, asimismo, la referida Ley N° 27.399 faculta al PODER EJECUTIVO NACIONAL a fijar anualmente hasta TRES (3) días feriados o no laborables destinados a promover la actividad turística, que deberán coincidir con los días lunes o viernes.

**Comment by martingra** [comment]
That's correct. There are up to 3 holidays added each year, depending on the holiday calendar and the political decision for that year. For 2024, the 3 additional holidays have been set for April 1st, June 21st, and October 11th, as you can see here in purple https://www.argentina.gob.ar/interior/feriados-nacionales-2024

**Comment by martingra** [comment]
Yes, it is true. These holidays are determined at the beginning of each year and there is no rule to anticipate them, but they are still legal holidays. If they are not included, then the information is not 100% accurate :(
